### PR TITLE
[Test] Add model speed and accuracy tests for RGCN

### PR DESCRIPTION
**Pull request by jermainewang**
## Description

- Add model speed and accuracy tests for RGCN
- Fix a bug when there are multiple `@utils.parametrize` decorators.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [x] Related issue is referred in this PR

### CODE REVIEW

File: benchmarks/benchmarks/model_acc/bench_rgcn.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+@utils.parametrize('data', ['aifb', 'mutag'])
+def track_acc(data):
+    # args
+    if data == 'aifb':
+        num_bases = -1
+        l2norm = 0.
+    elif data == 'mutag':
+        num_bases = 30
+        l2norm = 5e-4
+    elif data == 'am':
```
Comment (classicsong) [inline benchmarks/benchmarks/model_acc/bench_rgcn.py:57]: am is not in the dataset.

File: benchmarks/benchmarks/model_speed/bench_rgcn.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+@utils.parametrize('data', ['aifb', 'am'])
+def track_time(data):
+    # args
+    if data == 'aifb':
+        num_bases = -1
+        l2norm = 0.
+    elif data == 'am':
+        num_bases = 40
+        l2norm = 5e-4
+    elif data == 'mutag':
```
Comment (classicsong) [inline benchmarks/benchmarks/model_speed/bench_rgcn.py:49]: mutag is not used.

File: benchmarks/benchmarks/utils.py [hunk 3]
```diff
@@ -120,12 +129,21 @@ def setup_track_acc(*args, **kwargs):
 
 def parametrize(param_name, params):
```
Comment (classicsong) [inline benchmarks/benchmarks/utils.py:130]: Can you add some docstring here. Most developer will use parameterize.

**Review by classicsong — APPROVED** [review]
LGTM. Let some comments.

File: benchmarks/benchmarks/model_acc/bench_rgcn.py (hunk 1 above)
Comment (jermainewang) [reply benchmarks/benchmarks/model_acc/bench_rgcn.py:57]: Right now processing am dataset takes too long. I will add them back when we have properly setup the dataset pipeline.

**Pull request merged**